apps/lead/views.py

```diff
@@ -8,6 +8,7 @@
 from django.contrib.postgres.search import TrigramSimilarity
 from django.db import models, transaction
 
+from deep import compiler
```

Comment:
```suggestion
from deep import compiler  # noqa: F401
```

https://flake8.pycqa.org/en/3.7.9/user/error-codes.html